app/src/main/java/one/mixin/android/util/ShareHelper.kt

```diff
@@ -20,61 +22,50 @@ class ShareHelper {
             }
     }
 
-    fun generateForwardMessageList(intent: Intent): ArrayList<ForwardMessage>? {
+    fun generateForwardMessageList(intent: Intent): ArrayList<ForwardMessage<ForwardCategory>>? {
         val action = intent.action
         val type = intent.type
         if (action == null || type == null) {
             return null
         }
-        val result = arrayListOf<ForwardMessage>()
+        val result = arrayListOf<ForwardMessage<ForwardCategory>>()
+        // TODO handle */*
```

Comment:
Is it done?

Reply:
This doesn't affect current behavior and can be improved in the future commits.